configuration/debian/tedge_agent/postrm

```diff
@@ -24,16 +24,24 @@ purge_agent_directory() {
     fi
 }
 
+remove_operation_file() {
+    if [ -f "/etc/tedge/operations/c8y/c8y_SoftwareUpdate" ]; then
+        rm -rf /etc/tedge/operations/c8y/c8y_SoftwareUpdate
+    fi
+}
+
 case "$1" in
     purge)
         remove_tedge_agent_user
         remove_sudoers_file
+        remove_operation_file
         purge_agent_directory
     ;;
 
     remove)
         remove_tedge_agent_user
         remove_sudoers_file
+        remove_operation_file
```

Comment:
Good point. I need to remove it anyway as it should not be declared in the agent.
This is a tough one to be honest.
I put it here initially as it seemed agent was the right place, but if you think about it for a moment it is the wrong place because it is actually not agent but rather sm-mapper who adds support for this operation...